### PR TITLE
Adds config to use Kinesis Appender

### PR DESCRIPTION
**Pull request by kenoir**
Depends:
- https://github.com/guardian/grid-infra/pull/113
- https://github.com/guardian/machine-images/pull/40

Uses:
- https://github.com/guardian/kinesis-logback-appender

### CODE REVIEW

File: common-lib/src/main/scala/com/gu/mediaservice/lib/LogConfig.scala [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    val a = new KinesisAppender()
+    a.setContext(context)
+    a.setEncoder(encoder)
+    // TODO: Properly configure appender from properties!
+    a.start()
+    a
+  }
+
+  def init(config: CommonPlayAppConfig) = {
+    if(config.stage != "DEV") {
```
Comment (akash1810) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/LogConfig.scala]: Is `match` preferred over `if`?

**Comment by kenoir** [comment]
@sihil Going to wait on your :+1: to merge this and its dependants, mind having a look?

**Comment by theefer** [comment]
:+1:

**Comment by sihil** [comment]
Sorry - this got buried in my inbox.

I'd recommend getting the `app` value by reading the tags on your instance. Otherwise this is great :smile:

**Comment by kenoir** [comment]
Yup. I'd prefer that.

On Mon, 24 Aug 2015 at 11:35 Simon Hildrew <notifications@github.com> wrote:

> Sorry - this got buried in my inbox.
>
> I'd recommend getting the app value by reading the tags on your instance.
> Otherwise this is great [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/grid/pull/1093#issuecomment-134140996>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Pull request merged**